<launch>
    <!-- args --> 
    <!-- Behzad's workstation ip --> 
    <!-- arg name="host_ip" value="10.157.34.101" /--> 
    <arg name="host_ip" value="10.157.90.51" /> 
    <!-- Hasans's workstation ip --> 
    <!-- arg name="host_ip" value="10.157.34.208" /--> 

    <!-- setting DEBUG to false prevents publishing and subscribing to rviz /--> 
    <!-- also planning to prevent stdio stuff(if they really matter)--> 
    <arg name="DEBUG" value="true" /> 
    <param name="ros_DEBUG" type="bool" value="$(arg DEBUG)" />

    
    <!-- obj detection node --> 
    <param name="ip_addr" type="string" value="$(arg host_ip)" />
    <param name="detect_thresh" type="double" value=".4" />
    <node pkg="follow_the_leader" type="detection_node" name="detection_node"  output="screen">
    </node> 
    <!-- param name="mav_name" type="string" value="$(arg mav_name)" / -->
    <comment launch-prefix="gdb -ex run --args"/>


</launch>
